timezone/src/clients/timezone_proposal.rb

```diff
@@ -28,15 +26,7 @@ def main
         @force_reset = Ops.get_boolean(@param, "force_reset", false)
         @language_changed = Ops.get_boolean(@param, "language_changed", false)
 
-        @m = Convert.to_map(
-          SCR.Execute(path(".target.bash_output"), "/bin/date +%Y")
-        )
-        Builtins.y2milestone("date call: %1", @m)
-
-        if Ops.less_than(
-            Builtins.tointeger(Ops.get_string(@m, "stdout", "0")),
-            2004
-          )
+        if Time.now.year < 2015
```

Comment:
Yes, using a fixed date does not work well, this hasn't been updated since 2004...
I'll fix that.